client/src/App.js

```diff
@@ -33,14 +37,23 @@ const App = () => {
 		//  data: someData
 	}
 
-
+	const Routing = () => {
+		return (
+			<Routes>
+					{/* <Route exact path="/" element={<Home />} /> */}
+					<Route path="/settings" element={<SettingsPage/>}/>
+			</Routes>
+		)
+	}
 
 	return(
 		<div className="app-container">
 			<NewNavBar/>
 			<Game defaultSnippet={defaultSnippet.data}/>
```

Comment:
Game should have its own Route, although you'd have to implement the routing a little differently to achieve the same popup effect.

Reply:
Yes we agree, for now since settings is the only additional path we have, our implementation is acceptable. Once we begin work on account pages etc., we will take a closer look at this.